x-pack/plugins/index_management/public/application/components/mappings_editor/components/document_fields/fields/create_field/create_field.tsx

```diff
@@ -164,6 +172,10 @@ export const CreateField = React.memo(function CreateFieldComponent({
       </EuiFlexItem>
     </EuiFlexGroup>
   );
+  let wrapperClassName = 'mappingsEditor__createFieldWrapper ';
+  isAddingFields !== undefined && isAddingFields
+    ? (wrapperClassName += 'mappingsEditor__createFieldAddField')
+    : (wrapperClassName += 'mappingsEditor__createFieldBorder');
```

Comment:
Added this since we don't want form borders in create field form row in index Details page but in index/component templates.